app/src/main/java/org/fossasia/susi/ai/skills/skilldetails/SkillDetailsFragment.kt

```diff
@@ -112,8 +111,10 @@ class SkillDetailsFragment : Fragment(), ISkillDetailsView {
     private fun setImage() {
         skillDetailImage.setImageResource(R.drawable.ic_susi)
         if (skillData.image != null && !skillData.image.isEmpty()) {
-            Picasso.with(activity?.applicationContext).load(StringBuilder(imageLink)
-                    .append(skillGroup).append("/en/").append(skillData.image).toString())
+            Picasso.with(activity?.applicationContext)
+                    .load(StringBuilder(imageLink)
+                            .append("model=" + skillData.model + "&language=" + skillData.language + "&group=" + skillData.group.replace(" ", "%20"))
+                            .append("&image=").append(skillData.image).toString())
```

Comment:
Make it a function and reuse

Reply:
Yes, so make the function in a Utils file. Separate classes are not an excuse for duplication